fundamentals/fcsreqs.tex

```diff
@@ -26,6 +26,12 @@ \subsubsection{Discrete Facilities and Materials}
 
 \subsubsection{Open Access}
 
+% gonuke: I am perhaps oversensitive, but I perceive the status of 
+%         VISION to be constantly shifting.  I am thus unsure whether
+%         ``license agreemnts or instutional approval'' are currently
+%         required for access to VISION.  I raise it here to satisfy
+%         my desire to have it categorically confirmed by someone.
```

Comment:
This may be worth getting an update from INL about what is necessary for using VISION.